src/unstuck-bridge-tx.md

```diff
@@ -0,0 +1,286 @@
+# How to Unstuck SORA→ETHEREUM Transactions
+
+::: info
+This guide is written by the community member "Frasier Man" and reviewed by the SORA team. If you want thank Fraiser man, please donate to: `5D4pKyFNRjU8yoYKgiH1UqrjzayjCkttt82HsAdwQ6FTsQyv`
+:::
+
+### Use This Guide If:
+
+- Your SORA bridge transaction is stuck on pending or has failed.
+- The "retry" command on the bridge UI is not working/not available.
+- Your funds have left your SORA account and not arrived in your Ethereum account after some time.
+
+:::info
+If at the end of Step 4, Ethereum asks you for a LARGE
```

Comment:
Would be nice to include the direct links to the steps (that are the headings essentially)

Reply:
Yes, I was also considering implementing this